mesonbuild/interpreter/interpreter.py

```diff
@@ -677,42 +677,41 @@ def validate_arguments(self, args, argcount, arg_types):
                 if not isinstance(actual, wanted):
                     raise InvalidArguments('Incorrect argument type.')
 
-    @FeatureNewKwargs('run_command', '0.50.0', ['env'])
-    @FeatureNewKwargs('run_command', '0.47.0', ['check', 'capture'])
-    @permittedKwargs({'check', 'capture', 'env'})
-    # Executables aren'ta actually accepted, but we allow them here to allow for
-    # better error messages when overriden
+    # Executables aren't actually accepted, but we allow them here to allow for
+    # better error messages when overridden
     @typed_pos_args(
         'run_command',
         (build.Executable, ExternalProgram, compilers.Compiler, mesonlib.File, str),
         varargs=(build.Executable, ExternalProgram, compilers.Compiler, mesonlib.File, str))
+    @typed_kwargs(
+        'run_command',
+        KwargInfo('check', (bool, NoneType), since='0.47.0'),
+        KwargInfo('capture', bool, default=True, since='0.47.0'),
```

Comment:
This was added in #3413 without reference manual documentation. :(

Reply:
At least it'll be fixed now :D